scripts/k8s_deploy_monitoring.sh

```diff
@@ -26,7 +26,7 @@ fi
 # Deploy Monitoring stack
 helm status kube-prometheus >/dev/null 2>&1
 if [ $? -ne 0 ] ; then
-    helm install coreos/kube-prometheus --name kube-prometheus --namespace monitoring --values config.example/kube-prometheus.yml
+    helm install coreos/kube-prometheus --name kube-prometheus --namespace monitoring --values config.example/helm/kube-prometheus.yml
```

Comment:
It doesn't technically matter, but since `virtual/cluster_up.sh` is doing `cp -r config.example/ virtual/config/`, we should point at `config` instead of `config.example` since the former may have been modified.

Reply:
I did that because I was trying to mimic what regular (non-virtual) users would do - cp the config.example to config. Is that still the case?

This is modifying a file outside of virtual, which is why I ask.

Reply:
Oh sorry, didn't notice that. The reason I pointed to `config.example` instead of `config` was that in the K8s cluster deployment you don't need anything in `config`. It should probably check for `config` and use that if it exists, `config.example` otherwise.